src/control-plane/backend/lambda/api/service/stack-excution.ts

```diff
@@ -247,7 +248,7 @@ async function getIngestionBranch(pipeline: IPipeline, resources: CPipelineResou
     return undefined;
   }
   let ingestionTemplateKey = `${PipelineStackType.INGESTION}_${pipeline.ingestionServer.sinkType}`;
-  if (pipeline.ingestionServer.ingestionType === IngestionType.Fargate) {
+  if (SolutionVersion.Of(pipeline.templateVersion ?? FULL_SOLUTION_VERSION).greaterThanOrEqualTo(SolutionVersion.V_1_2_0)) {
```

Comment:
I think we still existing ingestion template for upgrading.

1.1.x -> 1.2  build a (use v1) -> 1.2 build b (use v1) -> 1.3 (use v1)
1.2 build a(use v2) -> 1.2 build b(use v2) -> 1.3(use v2)